virttest/migration.py

```diff
@@ -297,7 +297,10 @@ def do_migration(self, vms, srcuri, desturi, migration_type,
                         elif func == virsh.migrate_postcopy:
                             func(vm.name, uri=srcuri, debug=True)
                         else:
-                            func(args['func_params'])
+                            try:
```

Comment:
Is it better to firstly check if args['func_params'] is available instead of using try clause which is not good for performance.

Reply:
> @Yingshun , what is the original error without the fix?
it depends on the params. 
if there is no args['func_params'], it raise KeyError, else raise TypeError.

Reply:
But sometimes, people may set func_params incorrectly.